src/sass/mui/_select.scss

```diff
@@ -117,4 +117,16 @@ $xFormLabelLineHeight: floor($mui-label-font-size * 1.25);
       background-color: mui-color('grey', '200');
     }
   }
+
+  > .optgroup-option {
```

Comment:
By convention, MUI CSS classes use the "mui-" namespace and follow BEM syntax. Here are some suggestions for optgroup class names:
`.mui-optgroup__label`
`.mui-optgroup__option`